src/main/java/screens/NodeCenter.java

```diff
@@ -12,14 +12,23 @@
  * Implementation of the state that handles click events regarding the center query of selected node.
  */
 public class NodeCenter implements INodeHandler {
-    private GraphScene graphScene;
 
+    /**
+     * Screen that displays the graph.
+     */
+    private GraphScene graphScene;
 
+    /**
+     * Constructor.
+     * @param sc Scene for displaying graphs.
+     */
     public NodeCenter(GraphScene sc) {
         this.graphScene = sc;
 
     }
 
+
```

Comment:
Redundant empty line, and this method has no javadoc, I understand that it implements the handle method from the INodeHandler interface, however, the javadoc in that interface also doesn't properly describe what this implementation of the method does.